test/nls_testutils.jl

```diff
@@ -20,7 +20,7 @@
     check_nlp_dimensions(lls, exclude = [hess, hess_coord, jth_hess, jth_hess_coord, jth_hprod])
   end
   @testset "Multiple precision support" begin
-    multiple_precision_nls("LLS")
+    multiple_precision_nls(LLS)
```

Comment:
I don't think this is what we want. The old call was wrong as well (same with ADNLP). Calling "LLS" is actually calling the manual model from NLPModelsTest. We want that `LLSModel` above to be generalized to `T`.